deploy/manager/dev/50_manager_deployment.yaml

```diff
@@ -36,6 +36,13 @@ spec:
         volumeMounts:
         - mountPath: /mnt/etc/scylla-manager
           name: scylla-manager-config
+        readinessProbe:
+          httpGet:
+            path: /api/v1/clusters
+            port: 5080
+          initialDelaySeconds: 60
+          periodSeconds: 30
```

Comment:
my previous comment was meant for the exec. with http probe, we could do the 10s which we do for scyllaclusters

Reply:
changed to 10s and removed timeout